Billing.cs

```diff
@@ -1 +1,208 @@
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Data.SqlClient;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Library
+{
+    public partial class Billing : Form
+    {
+        public Billing()
+        {
+            InitializeComponent();
+            populate();
+        }
+
+        private void label4_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void guna2DataGridView1_CellContentClick(object sender, DataGridViewCellEventArgs e)
+        {
+
+        }
+        SqlConnection Con = new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=C:\Users\Margar\Documents\LibraryDatabase.mdf;Integrated Security=True;Connect Timeout=30");
+        private void populate()
+        {
+            Con.Open();
+            string query = "select * from BookTable";
+            SqlDataAdapter sda = new SqlDataAdapter(query, Con);
+            SqlCommandBuilder builder = new SqlCommandBuilder(sda);
+            var ds = new DataSet();
+            sda.Fill(ds);
+            BookGrid.DataSource = ds.Tables[0];
+            Con.Close();
+        }
+        private void SaveBtn_Click(object sender, EventArgs e)
+        {
+            int n = 0;
+            //if()
+            DataGridViewRow newRow = new DataGridViewRow();
+            newRow.CreateCells(BillGrid);
+            newRow.Cells[0].Value = n + 1;
+            newRow.Cells[1].Value = UsrName.Text;
+            newRow.Cells[2].Value = PriceTb.Text;
+            BillGrid.Rows.Add(newRow);
+            n++;
+        }
+
+        private void panel1_Paint(object sender, PaintEventArgs e)
+        {
+
+        }
+        int key = 0;
+        private void BillGrid_CellContentClick(object sender, DataGridViewCellEventArgs e)
+        {
+            UsrName.Text = BookGrid.SelectedRows[0].Cells[1].Value.ToString();
+            //Quan.Text = BillGrid.SelectedRows[0].Cells[4].Value.ToString();
+            //ClientNameTb.Text = BillGrid.SelectedRows[0].Cells[2].Value.ToString();
+
+            PriceTb.Text = BookGrid.SelectedRows[0].Cells[4].Value.ToString();
+            if (UsrName.Text == "")
+            {
+                key = 0;
+            }
+            else
+            {
+                key = Convert.ToInt32(BookGrid.SelectedRows[0].Cells[0].Value.ToString());
+            }
+        }
+
+        private void label1_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void Reset()
+        {
+            UsrName.Text = "";
+            PriceTb.Text = "";
+            ClientNameTb.Text = "";
+
+        }
+        private void ResetBtn_Click(object sender, EventArgs e)
+        {
+            Reset();
+        }
+
+        private void ClientNameTb_TextChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void label9_Click(object sender, EventArgs e)
```

Comment:
Label could be meaningfully named
```suggestion
        private void exit_Click(object sender, EventArgs e)
```